packages/visualizations/addon/components/number-format-selector.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *  {{number-format-selector
+ *    format=format
+ *    onUpdateFormat=onUpdateFormat
+ *  }}
+ */
+
+import { get, computed } from '@ember/object';
+import { A } from '@ember/array';
+import Component from '@ember/component';
+import layout from '../templates/components/number-format-selector';
+
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['number-format-selector'],
+
+  /**
+   * @property {Array} predefinedFormats
+   * list of format types shown to the user
+   */
+  predefinedFormats: [
+    { name: 'Number', format: '0,0.00', class: "number-format-selector__radio-number" },
```

Comment:
Do we need to maintain the class property? Can we make use of the name in the template?

`<div class={{concat 'number-format-selector__radio-' predefinedFormat.name}}>`